packages/stellar-axelar-std-derive/src/contractimpl.rs

```diff
@@ -35,6 +44,57 @@ pub fn contractimpl(impl_block: &mut ItemImpl) -> Result<proc_macro2::TokenStrea
     })
 }
 
+fn all_contract_endpoints(
```

Comment:
I don't think we need to make a distinction between all_contract_endpoints and any_stateful_endpoints. This function also doesn't return all, just the ones with env in the parameter list

Reply:
I suggest going with the existing `has_args` check, because you're parsing for `env` later anyway